### ISSUE TITLE
False positive for `logging-fstring-interpolation`

### ISSUE THREAD

**Issue by yegarti** [PyCQA/pylint#4984]
### Bug description

When using f-string without arguments in a logging message, Pylint would warn me about not using lazy % formatting, even though I do use it.

a.py:
```python
import logging
logging.error(f'Hello %s', 'World!')
```

output:
`ERROR:root:Hello World!`

### Command used

```shell
pylint a.py
```

### Pylint output

```shell
************* Module a
a.py:2:0: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)
a.py:2:14: W1309: Using an f-string that does not have any interpolated variables (f-string-without-interpolation)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior

Report only `f-string-without-interpolation`

### Pylint version

```shell
pylint 2.10.2
astroid 2.7.3
Python 3.9.4 (default, Apr 30 2021, 18:51:54) 
[GCC 10.2.0]
```

### OS / Environment

Manjaro Linux, Kernel 5.10

**Comment by sergeyklay** [comment]
Possible related https://github.com/PyCQA/pylint/issues/5039#issuecomment-925126649
